src/app/share-report/share-report.component.ts

```diff
@@ -368,24 +371,51 @@ export class ShareReportComponent implements AfterViewInit {
           if (result) {
             this.actionService.startAction(ReportAction.MUTE_TWITTER);
             await firstValueFrom(
-              this.twitterApiService.muteUsers(this.getUsersInReport())
+              this.twitterApiService.muteUsers(this.getTwitterUsersInReport())
             )
               .then((response) => {
-                const failures = response.failedScreennames;
-                if (failures?.length) {
+                const numQuotaFailures = response.numQuotaFailures;
+                const numOtherFailures = response.numOtherFailures;
+                if (numQuotaFailures) {
+                  // Some of the errors were quota issues, so we combine those
+                  // with any other issues to keep things straightforward for
+                  // users.
+                  const numFailures =
+                    numQuotaFailures + (numOtherFailures ?? 0);
                   this.dialog.open(ApiErrorDialogComponent, {
                     panelClass: 'api-error-dialog-container',
                     data: {
-                      failures,
                       message:
-                        'The following users could not be muted. These accounts ' +
-                        'may no longer be active or an unknown error may have ' +
-                        'occurred.',
-                      title: `${failures.length} ${
-                        failures.length === 1 ? 'user' : 'users'
+                        "Only 50 users can be muted every 15 minutes. If you'd " +
```

Comment:
Nit: Be consistent about " and '. If the issue is with "you'd", then you can escape the ' in the string.

Reply:
Done.